portal/src/main/webapp/sitemap_index.jsp

```diff
@@ -0,0 +1,30 @@
+<%@ page trimDirectiveWhitespaces="true" %>
+<%@ taglib prefix = "c" uri = "http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/xml; charset=UTF-8" %>
+<%@ page import="org.json.simple.JSONArray" %>
+<%@ page import="org.json.simple.JSONObject" %>
+<%@ page import="org.json.simple.parser.*" %>
+<%@ page import="java.util.*" %>
+<%@ page import="org.mskcc.cbio.portal.util.GlobalProperties" %>
+
+<%
+String protocol = (request.isSecure()) ? "https" : "http";
```

Comment:
somehow doesn't seem to list correct protocol: https://cbioportal-pr-4338.herokuapp.com/sitemap_index.xml . I see http instead of https